posthog/api/test/test_event.py

```diff
@@ -21,6 +21,16 @@ def test_filter_events(self):
         self.assertEqual(response['results'][0]['person'], 'tim@posthog.com')
         self.assertEqual(response['results'][0]['elements'][0]['tag_name'], 'button')
 
+    def test_filter_events_by_event_name(self):
+        person = Person.objects.create(properties={'email': 'tim@posthog.com'}, team=self.team, distinct_ids=["2", 'some-random-uid'])
+        event1 = Event.objects.create(event='event_name',team=self.team, distinct_id="2", properties={"$ip": '8.8.8.8'}, elements=[
```

Comment:
Only `$autocapture` events have elements, so just to avoid confusion would you mind removing `elements=[...]`